review/fileToReview.js

```diff
@@ -0,0 +1,73 @@
+
+var domain = "bank.local.fr"
+
+/**
+ * @description Fetch transactions recursively
+ * @param {string} fromDate The maximum date of transactions to return
+ * @param {string} authorization Authorization header to authent the user
+ * @param {jws} jws Jws token, mandatory if we get one from login request
+ * @param {Number} id Account id
+ * @param {Number} page Number of the page
+ * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
+ * @return {Object} All transactions available on the page
+ */
+async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
```

Comment:
Je propose de changer certains noms de paramètres pour une meilleure compréhension, si c'est la date max, j'imagine que c'est plus un `toDate` :
```suggestion
 * @param {string} toDate The maximum date of transactions to return
 * @param {string} authorization Authorization header to authenticate the user
 * @param {string} jwsToken Jws token, mandatory if we get one from login request
 * @param {Number} accountId Account id
 * @param {Number} pageNumber Number of the page
 * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
 * @return {Object} All transactions available on the page
 */
async function fetchTransactions(toDate, authorization, jwsToken = null, accountId, pageNumber, previousTransactions) {
```